pkg/kubelet/cri/remote/remote_runtime_test.go

```diff
@@ -67,7 +67,8 @@ func TestVersion(t *testing.T) {
 
 	r := createRemoteRuntimeService(endpoint, t)
 	version, err := r.Version(apitest.FakeVersion)
-	assert.NoError(t, err)
-	assert.Equal(t, apitest.FakeVersion, version.Version)
-	assert.Equal(t, apitest.FakeRuntimeName, version.RuntimeName)
+	if assert.NoError(t, err) {
```

Comment:
Rather than putting this in an if statement, if you switch to `require.NoError(t, err)`, (and `require.Equal`...),  then the tests will fail early without needing to add branching.

Reply:
let me make the change

Reply:
@endocrimes I have changed to use require.NoError.